nextjs-13/src/lib/handler/base.handler.ts

```diff
@@ -0,0 +1,37 @@
+import { User } from '@prisma/client';
+import { NextResponse, NextRequest } from 'next/server';
+import { getToken } from 'next-auth/jwt';
+
+import { findUserById } from 'repositories/user.repository';
+
+export default async function baseHandler(
+  req: NextRequest,
+  callback: (currentUser: User, body: unknown) => Promise<NextResponse>
+) {
+  try {
+    const token = await getToken({ req });
+
+    if (token && token.userId) {
+      const currentUser = await findUserById(token.userId as string);
+
+      if (currentUser) {
+        let body = {};
+
+        try {
+          body = await req.json();
+        } catch (err) {}
+
+        console.log('========>body : ', body);
```

Comment:
Maybe we shouldn't leave the console.log in the code base here, lets confirm it with @olivierobert I guess.

Reply:
Oh I deliberately want to leave this here so that I can view all the bodies of the requests 👀

Reply:
Ye I think so. In ruby, the equivalent of `console.log` is `print`, but when people want to log things in production, they use [`Logger`](https://ruby-doc.org/stdlib-2.4.0/libdoc/logger/rdoc/Logger.html) instead. So I think maybe we only use `console.log` at the local 👀

Reply:
Noted, Thanks for the knowledge 😆

Reply:
This should be removed. This is fine during development but for production load, we normally https://github.com/winstonjs/winston or https://github.com/pinojs/pino

Reply:
The removed commit: https://github.com/nimblehq/js-framework-benchmark/pull/96/commits/455c612b291d7e0f3ddf58b4f2b24ae0f3e08756#diff-10b91e667909ffe6c8b2d34d735e0fa46be166ba37cf1b5ec24f7ea6b54658bbL24